plugins/providers/oss/config.go

```diff
@@ -0,0 +1,95 @@
+package oss
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/goto/guardian/domain"
+	"github.com/goto/guardian/utils"
+	"github.com/mitchellh/mapstructure"
+)
+
+const (
+	AccountTypeRAMUser  = "ram_user"
+	AccountTypeRAMRole  = "ram_role"
+	resourceTypeBucket  = "bucket"
+	parameterRAMRoleKey = "ram_role"
```

Comment:
since the value is similar, can't we just use `AccountTypeRAMRole` ?

Reply:
this field is not being used, I've removed it for now. thanks for pointing out!